receiver/sqlqueryreceiver/logs_receiver.go

```diff
@@ -291,7 +291,10 @@ func (queryReceiver *logsQueryReceiver) collect(ctx context.Context) (plog.Logs,
 		rows, err = queryReceiver.client.QueryRows(ctx)
 	}
 	if err != nil {
-		return logs, fmt.Errorf("error getting rows: %w", err)
+		if !errors.Is(err, sqlquery.ErrNullValueWarning) {
+			return logs, fmt.Errorf("error getting rows: %w", err)
+		}
+		queryReceiver.logger.Warn("problems encountered getting log rows", zap.Error(err))
```

Comment:
I don't know what the actual log message will be here, but since we know it's a null value error, the default statement could specifically mention that a null value has been encountered.